src/Data/Arrays.php

```diff
@@ -264,6 +262,8 @@ public static function multiToAssocWithSpecificOpr(array $arrays, $opr = null)
             if (self::isReallyArray($value) === true) {
                 $results = array_merge($results, self::multiToAssocWithSpecificOpr($value, $key.$opr));
             } else {
+                $key = $opr.$key;
+                $key = ($key[0] === '.' || $key[0] === '@' ? substr($key, 1) : $key);
```

Comment:
This is about fixing customized operators quickly.

Perhaps we should define an operator lists and using the `in_array` function to validate `$opr`.